pkg/controller/workspace/status.go

```diff
@@ -35,8 +35,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	//	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	//	logf "sigs.k8s.io/controller-runtime/pkg/runtime/Log"
```

Comment:
Find/replace accidental change?

Reply:
Good catch. I believe such commented import can be removed now.